lib/Player.tsx

```diff
@@ -254,16 +254,13 @@ export const Player = forwardRef<PlayerNativeElement, PlayerProps>(
     }, [videoProperties, volume])
 
     /**
-     * Refresh on RTCP bye
+     * Refresh on stream end
```

Comment:
I guess we only want to refresh when the server ended the stream, in which case we should state that here?

Reply:
Yes, this is only when the server ended the stream. We didn't really deal with this before, at least not on this level.